setup.py

```diff
@@ -4,8 +4,22 @@
 """The setup script."""
 
 from setuptools import find_packages, setup
+from setuptools.command.build_py import build_py
+from pathlib import Path
 from typing import Dict, List
 
+
+class BuildPyCommand(build_py):
+    """Check for existence of XSLT before building."""
+    def run(self):
+        xslt = Path(__file__).parent / "aicsimageio/metadata/czi-to-ome-xslt/xslt/czi-to-ome.xsl"
+        if not xslt.is_file():
+            raise FileNotFoundError(
+                "XSLT not found. Is the submodule checked out?"
+            )
+        build_py.run(self)
```

Comment:
Is there a way to add a test to the CI that builds the wheel and then installs from the wheel and then tests just like the metadata tests -- from the install? Seems complicated but would be lovely as a sanity check because we are getting into the wonderful world of python packaging.

Reply:
That's what `tox` does! It took me a bit to make sure it was working, but it definitely seems to be now.

Reply:
Tox is great. All hail tox.